tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>If you think all of this info is useful, feel free to keep it. I would consider "demonstrate its effectiveness in reducing the model size after applying zip while keeping decent accuracy" to be redundant with the summary at the top and "keeping decent accuracy" to be redundant with "For this example, there is minimal loss in test accuracy after clustering, compared to the baseline." below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Does the tensorflow documentation style guide require minimum description in the notebook? If so I will delete those, and if not, I will keep it since it can guide the reader on what to expect in the section.</p>